src/elm/Avatar.elm

```diff
@@ -36,8 +36,16 @@ encode (Avatar maybeHash) =
             Encode.null
 
 
-view : String -> Avatar -> String -> Html msg
-view url (Avatar maybeHash) cls =
+view : String -> Avatar -> String -> String -> Html msg
+view url (Avatar maybeHash) username cls =
+    Html.a
+        [ href ("/profile/" ++ username)
+        ]
+        [ viewIcon url (Avatar maybeHash) cls ]
+
+
+viewIcon : String -> Avatar -> String -> Html msg
+viewIcon url (Avatar maybeHash) cls =
```

Comment:
What does cls stands for?

Reply:
Classes. It's how the original code called that parameter so I decided to use it as-is.

Reply:
Ohhhh now I see it, you added the link to the avatar module, Ill have to review it in a little more depth.

Reply:
No problem :)

Reply:
Hey @RodrigoLeiteF you added only a link to the avatar, which is not the right way, since when clicking the name will not work.

Also we are doing a link to profile, so it makes sense that it goes into the `Profile.elm` module. 

It removes the necessity of adding an extra param to the avatar module (if you think about it the avatar should only handle avatar, right? So the profile shouldn't be something the avatar should be concerned for)

That being said, you should move this functionality from the `Avatar.elm` module, and add it to the `Profile.elm` module.